assets/js/form-type-text-editor.js

```diff
@@ -48,6 +48,12 @@ document.addEventListener('DOMContentLoaded', () => {
         trixContentElement.removeAttribute('required');
     });
 
+    // Because of the way TrixEditor works, the jquery.are-you-sure plugin cannot detect changes to these fields automatically,
+    // so we manually trigger the plugin when the content changes.
+    document.addEventListener("trix-change", function (event) {
+        $(event.target).parents("form").trigger("checkform.areYouSure");
```

Comment:
```suggestion
    document.addEventListener('trix-change', function (event) {
        $(event.target).parents('form').trigger('checkform.areYouSure');
```
Maybe switch to single quotes, like the rest of the file?

Reply:
Also, I'm not sure, but maybe `closest()` should be used instead of `parents()`?

Reply:
I agree, `closest()` is better